tests/std/tests/GH_002581_common_reference_workaround/test.cpp

```diff
@@ -0,0 +1,42 @@
+#include <cstddef>

```

Comment:
you forgot the comment:
```c++
// Copyright (c) Microsoft Corporation.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Also we should modify the file:
https://github.com/microsoft/STL/blob/ff5445078a73b5e9445dffc656028a47fa790509/tests/std/test.lst

Reply:
Thanks! I'll add to these files when sufficient workarounds are done.